app/utils/groupme/Bot.scala

```diff
@@ -0,0 +1,21 @@
+package utils.groupme
+
+import play.api.Logger
+import play.api.Play.current
+import play.api.libs.concurrent.Execution.Implicits._
+import play.api.libs.json.{Json,JsValue}
+import play.api.libs.ws.WS
+
+class Bot(idc: String, urlc: String) {
+  var id: String = idc
+  var url: String = urlc
+
+  def post(text: String) = {
+    val data: JsValue = Json.obj("bot_id" -> id, "text" -> text)
+    WS.url(url).post(data).map {
+      case response => {
+        // Log if response.status is not 2xx?
```

Comment:
Just logging makes sense to me. We don't want to return a 400 or anything to the 151er. Maybe bubble up an exception so that the redirect to "/displayEntries" can display that we couldn't post to GroupMe... but eh, we can do that later.

Bryan was going to work on making the view/edit entries appear on the same page as where you record new entries. When we do that we could have an error pane or popup to let the user know it was added but not posted.